pkg/cmd/roachtest/roachtestutil/utils.go

```diff
@@ -215,3 +217,19 @@ func IfLocal(c cluster.Cluster, trueVal, falseVal string) string {
 	}
 	return falseVal
 }
+
+// CheckPortBlocked returns true if a connection from a node to a port on another node
+// can be established. Requires nmap to be installed.
+func CheckPortBlocked(
+	ctx context.Context, l *logger.Logger, c cluster.Cluster, fromNode, toNode option.NodeListOption, port string,
```

Comment:
Nit: typically `port` is a (positive) `int`; the weaker type could lead to some typos or API abuse.

Reply:
Hmm, I intentionally left it as a string so you could take advantage of roachprod expanders, e.g. `{pgport:1}`. Not that there's anything wrong with calling 
```
ports, err := `c.SQLPorts()`
srcPort := ports[0]
...
```

but seemed like extra boilerplate to me. Don't feel strongly about it though, happy to switch to an int if we prefer.